src/store/reducers/dform/initial/styles.json

```diff
@@ -1317,6 +1317,88 @@
               "id": 0,
               "label": "कहिल्यै पनि भएन (0)"
             }
+          ] 
+        },
+        {
+          "field": "dep",
```

Comment:
Sorry, I didn't told but please note that `field` is not necessary for `"class": "block"` because `Block` is just a layout component and does not represent any field of the record.

Reply:
Noted that field refers to record fields